### PR TITLE
Fixed Button Expansion Issue

### PR DESCRIPTION
**Pull request by nikldev0**
For "Copy button expanding on clicking #303"

### CODE REVIEW

**Comment by daltonfury42** [comment]
@nikldev0 I think we should rather fix the size of the Tick mark icon

**Comment by nikldev0** [comment]
> 
> 
> @nikldev0 I think we should rather fix the size of the Tick mark icon

Sure @daltonfury42 let me change that and revert back to the min-height property

**Comment by nikldev0** [comment]
@daltonfury42 done thanks, pls check

File: simplq/src/styles/buttons.module.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   background-color: $primary-color-dark;
   color: #fff;
   display: inline-block;
-  padding: 0.2em 0.5em 0.3em 0.5em;
+  padding: 0.2em 0.5em 0.3em 0.2em;
```
Comment (daltonfury42) [inline simplq/src/styles/buttons.module.scss:7]: Why did we reduce this padding?

Comment (nikldev0) [reply simplq/src/styles/buttons.module.scss:7]: I thought it looked a little odd with the additional space to the left of the tick icon so I reduced it, but will revert it if you would like?

Comment (daltonfury42) [reply simplq/src/styles/buttons.module.scss:7]: I see, let's see how it looks.

File: simplq/src/styles/buttons.module.scss [hunk 2]
```diff
@@ -29,6 +29,10 @@
   }
 }
 
+.standard-button img {
+  height: 24px;
```
Comment (daltonfury42) [inline simplq/src/styles/buttons.module.scss:33]: Can you make this `1.5rem`? We prefer rem over px

File: simplq/src/styles/buttons.module.scss [hunk 3]
```diff
@@ -56,6 +60,9 @@
   }
 }
 
+
+
+
```
Comment (daltonfury42) [inline simplq/src/styles/buttons.module.scss:65]: remove

**Review by daltonfury42 — APPROVED** [review]

**Comment by daltonfury42** [comment]
Thanks for the PR!

**Pull request merged**